docs/semgrep-app/findings.md

```diff
@@ -39,48 +63,54 @@ The Semgrep App Finding page displays findings across all projects connected to
+| **Rule**        | The name of the rule that  matched with the code. Clicking this field brings you to the rule’s entry in the [Rule Registry](https://semgrep.dev/r). |
+| **Introduced**  | The time when this finding was first found. Clicking the header row sorts the table by latest or earliest findings. |
+| **Status**      | Refers to a finding’s triage state. Refer to the following table to understand triage states. |
 
 ## Triaging findings
 
-Triaging means prioritizing a finding based on a policy or criteria set by your team or organization. While severity is a factor in triage, your organization may define additional criteria based on coding standards, business, or product goals.
+**Triaging** means prioritizing a finding based on a policy or criteria set by your team or organization. While severity is a factor in triage, your organization may define additional criteria based on coding standards, business, or product goals.
 
 Semgrep App assists in the triage process through the use of **comments** and **triage states**:
 
 | Triage state | Description |
 | -----------  | ------------ |
 | **Open** | Open findings require action, such as rewriting the code for vulnerabilities, or refactoring the code. Findings are open by default. |
-| **Ignored** | Findings that are ignored will not be acted upon. This may be a false positive or deprioritized issue. Findings can be ignored through Semgrep App (see [Managing triage states](#managing-triage-states-bulk-triage)). |
-| **Closed** | Findings are considered closed under any of the following conditions: <ul><li>After a new scan, the code has been addressed such that the rule pattern no longer matches the code.</li><li>The rule has been removed from your Rule Board.</li></ul> |
+| **Ignored** | Findings that are ignored will not be acted upon. This may be a false positive or deprioritized issue. Findings can be ignored through Semgrep App (see [Managing triage states](#managing-triage-states-bulk-triage)). There are two types of ignored findings: <dl><dt>Ignored - App</dt><dd>The ignore status was set within Semgrep App.</dd><dt>Ignored - Code</dt><dd>The ignore status was set from the code itself. Using a `nosemgrep` annotation within the code falls under this type.</dd></dl> |
+| **Fixed** | Fixed findings are findings that were detected in a previous scan but no longer trigger a match in the most recent scan. The rule that detected the finding and the code that triggered the match must both be active in the most recent scan. |
+
+Findings can also be **removed**. A removed finding does not count towards a fix rate or the total number of findings. A finding is considered removed if it is not found in the most recent scan due to either of the following conditions:
+
+* You removed the rule from the Rule Board.
+* You deleted the file containing the code.
+* The file is included in a `.semgrepignore` file.
```

Comment:
Could it also be that a //semgrepignore comment was added to (above?) the matching line?

Reply:
(I assume you mean the comment annotation `nosemgrep`.) 

There is actually a difference in ignoring via comment (`// nosemgrep`) and ignoring via file. //nosemgrep is considered a triage action, so Semgrep still scans the code, makes the finding record, and ignores it. It appears as "Ignored - Code" in the findings page. For that reason, I don't include adding a 'nosemgrep' comment to the code.

I did bring up before (a long time ago) that there's one word (ignore) for two different things. Maybe "muted" is a term we'd like to reexplore, but right now, this is how the product works.

Reply:
Yes, I meant `// nosemgrep` :)

Reply:
(Resolving since for now //nosemgrep does not do the same thing as adding a file in .semgrepignore :) )